### PR TITLE
added dist to android in ci pipeline

### PR DESCRIPTION
**Pull request by mihaiplesa**
## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [x] Android
  - [x] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [x] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -146,6 +146,11 @@ pipeline {
                                 sh "npm run build -- ${BUILD_TYPE} --channel=${CHANNEL} --target_os=android --target_arch=arm"
                             }
                         }
+                        stage("dist") {
+                            steps {
+                                sh "npm run create_dist -- ${BUILD_TYPE} --channel=${CHANNEL} --target_os=android --target_arch=arm"
```
Comment (bridiver) [inline Jenkinsfile]: I think we also need `android_override_version_name` here and build - https://github.com/brave/brave-browser/pull/7828/files#diff-1bb8ffaf71730e8067757db1a6a153dbR76

Comment (bridiver) [reply Jenkinsfile]: @SergeyZhukovsky can provide more details about the value for it

Comment (SergeyZhukovsky) [reply Jenkinsfile]: we still need to provide `android_override_version_name` that is taken from `package.json` desktop file I think. `android_override_version_name` is only a command line option

Comment (SergeyZhukovsky) [reply Jenkinsfile]: we fixed it on brave-browser side

**Pull request merged**